utils/reportsender.go

```diff
@@ -0,0 +1,62 @@
+package utils
+
+import (
+	"time"
+)
+
+type dummyReportSender struct{}
```

Comment:
This code should be in the `backend` package

Reply:
https://github.com/kubescape/backend/pull/14

Reply:
We merged the PR.